contentcuration/contentcuration/frontend/channelEdit/components/edit/EditView.vue

```diff
@@ -239,6 +252,15 @@
       trackTab(name) {
         this.$analytics.trackClick('channel_editor_modal', name);
       },
+      /*
+       * @public
+       * reaches into Details Tab to run save of diffTracker
+       * before the validation pop up is executed
+       */
+      immediateSaveAll: function() {
+        this.$refs.detailsTab.immediateSaveAll();
```

Comment:
Should return here, rather than returning a dummy promise on the next line here - otherwise we are still not waiting on the completion of `immediateSaveAll`